aries-site/src/pages/templates/forms.mdx

```diff
@@ -207,6 +208,27 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
   <CustomizeExample />
 </Example>
 
+### Character counter
+
+**When to use**  
+In cases where the user is given an opportunity to express themselves or would likely exceed a field's character limit. Examples include some description fields, feedback, feature requests, reporting issues, etc.  
+
+**Guidance**
+- Place the character counter at the bottom right of the field.  
+- [See guidance on character limits for fields](https://design-system.hpe.design/components/tag#anatomy). If you have doubts about how many characters to limit in the field, set a generous limit so as to not frustrate the user. When possible, check analytics to see how long it is taking your users to complete this task. Longer than usual time could indicate the character limits are too few. Increase accordingly.
```

Comment:
Sometimes the character count is limited by the backend -- wondering if we should mention that

Reply:
Better? I figure if the backend has limitations, maybe it's for security or data volume purposes and we shouldn't encourage them to mess with that LOL but you tell me. 

"Note that your backend may have character limitations. When you can choose the character limits, however, make it generous to avoid frustrating the user."

Reply:
Perfect.